languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/spelling.txt

```diff
@@ -59530,7 +59531,6 @@ Klingenkrümmung
 Klingenkrümmungen
 Monokini/S
 Jazzpants
-Shampoon/S
```

Comment:
added to `ignore.txt` so that it will not be offered as a suggestion -> I will add a style rule for these variants